vars/withParameterizedInfraPipeline.groovy

```diff
@@ -6,10 +6,10 @@ import uk.gov.hmcts.contino.MetricsPublisher
 import uk.gov.hmcts.contino.Subscription
 import uk.gov.hmcts.pipeline.TeamConfig
 
-def call(String product, String environment, String subscription, Closure body) {
-  call(product, environment, subscription, '', body)
+def call(String product, String environment, String subscription, Boolean planOnly, Closure body) {
```

Comment:
this is a breaking change and needs a new method without plan only added and passing false through in the `call` function

Reply:
fair enough